lend/src/utils/blockies.js

```diff
@@ -0,0 +1,108 @@
+// The random number is a js implementation of the Xorshift PRNG
```

Comment:
why not just import from `shared/` instead?

Reply:
removed some utils